docker-compose.yml

```diff
@@ -128,7 +128,7 @@ services:
   # CAdvisor container allows monitoring other containers. Useful for
   # development.
   cadvisor:
-    image: google/cadvisor:latest
+    image: gcr.io/cadvisor/cadvisor:latest
```

Comment:
We were using a very old version of cadvisor that crashed every time when running `docker compose up`.